core/ui/compose/designsystem/src/main/kotlin/app/k9mail/core/ui/compose/designsystem/molecule/input/CheckboxInput.kt

```diff
@@ -29,13 +31,18 @@ fun CheckboxInput(
         Row(
             modifier = Modifier
                 .fillMaxWidth()
-                .clickable { onCheckedChange(!checked) },
+                .semantics(mergeDescendants = true) {}
```

Comment:
This doesn't seem to be necessary for it to work on my test device. Does it make any difference for you?

Reply:
It did at some point, but now I don't see a difference.